lib/store_base_sti_class_for_5_0.rb

```diff
@@ -346,6 +346,24 @@ def build_through_record(record)
     end
 
     module Reflection
+      class PolymorphicReflection < ThroughReflection
```

Comment:
@n-rodriguez tests pass with the change above, please amend your commit with:

```
class PolymorphicReflection < AbstractReflection
``` 
Looks good on 5.0.0.1 as well as 5.0.1